website/addons/wiki/shareServer.js

```diff
@@ -60,31 +60,16 @@ app.use(morgan('common'));
 
 // Allow CORS
 app.use(function(req, res, next) {
-    res.header('Access-Control-Allow-Origin', settings.corsAllowOrigin);
+    res.header('Access-Control-Allow-Origin', 'http://localhost:5000');
     res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
+    res.header('Access-Control-Allow-Credentials', true);
```

Comment:
I think this line can be removed, you should not need to send the cookie credentials.